flask_security/core.py

```diff
@@ -98,6 +98,7 @@
     'CONFIRM_EMAIL_WITHIN': '5 days',
     'RESET_PASSWORD_WITHIN': '5 days',
     'LOGIN_WITHOUT_CONFIRMATION': False,
+    'LOGIN_SHOW_USER_EXISTENCE': True,
```

Comment:
Can you add a documentation?

https://github.com/mattupstate/flask-security/blob/7028e41cadf74b86ed5f86c7946fa3b191c07f7d/docs/configuration.rst

Reply:
updated